INFO     backend.layers.business.business:business.py:553 {'message': 'ingesting dataset', 'collection_version_id': 200bbcbf-2487-4680-9890-1b00a48e3fa6, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:553 {'message': 'ingesting dataset', 'collection_version_id': 200bbcbf-2487-4680-9890-1b00a48e3fa6, 'url': None, 'current_dataset_version_id': e6669fa2-683d-4d89-83b4-7b873a67e4f0}
INFO     processing:dataset_metadata_update.py:333 Main: No raw h5ad update required
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [4de2b4b9-7618-4ae1-96a7-34b159c736e4/raw.h5ad] to s3://artifact_bucket/4de2b4b9-7618-4ae1-96a7-34b159c736e4/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
ERROR    processing:dataset_metadata_update.py:353 Cannot find labeled H5AD artifact uri for e6669fa2-683d-4d89-83b4-7b873a67e4f0.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:357 Main: Starting thread for rds update
INFO     processing:dataset_metadata_update.py:380 Main: Starting thread for cxg update
INFO     processing:process_logic.py:45 Updating processing status